### ISSUE TITLE
filepaths in load directives should be OS agnostic

### ISSUE THREAD

**Issue by cari-lynn** [k14s/ytt#175]
When loading a library on windows with a file path with forward slashes you get a syntax error. For example, given a `lib\helpers.star`, `#@ load("lib/helpers.star", "...")` should work.

Currently there is an error:
```
- cannot load lib/helpers.star: Expected to find file 'lib/helpers.star' (hint: only files included via -f flag are available)
    in <toplevel>
      config.yml:6 | #@   load("lib/helpers.star", "testfunc")
```

**Comment by ewrenn8** [comment]
fixed in a6f62c85e591fa063ef50da43381c7787f628f0b

**Comment by cppforlife** [comment]
included in https://github.com/k14s/ytt/releases/tag/v0.30.0
